testsCommon/batchValidatorStub.go

```diff
@@ -0,0 +1,26 @@
+package testsCommon
+
+import (
+	"context"
+	"errors"
+
+	"github.com/ElrondNetwork/elrond-eth-bridge/clients"
+)
+
+// BatchValidatorStub -
+type BatchValidatorStub struct {
+	ValidateBatchCalled func(ctx context.Context, batch *clients.TransferBatch) (bool, error)
+}
+
+// ValidateBatch -
+func (bvs *BatchValidatorStub) ValidateBatch(ctx context.Context, batch *clients.TransferBatch) (bool, error) {
+	if bvs.ValidateBatchCalled != nil {
+		return bvs.ValidateBatchCalled(ctx, batch)
+	}
+	return false, errors.New("method not implemented")
```

Comment:
I would let it as it is, since with your suggestion the stub will behave as a disabledBatchValidator, without explicitly define it like that in the tests